### PR TITLE
Improve the fault tolerance doc

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

Improve the fault tolerance doc.

The original PR: https://github.com/tikv/website/pull/246

### CODE REVIEW

**Comment by marsishandsome** [comment]
@iosmanthus PTAL

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 1]
```diff
@@ -45,40 +51,41 @@ To view the Grafana: http://127.0.0.1:3000
...
+### Step 2. Import data to TiKV
 
-On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload of writing data to the TiKV cluster.
 
-1. Clone the `go-ycsb` from GitHub.
+1. Clone `go-ycsb` from GitHub.
 
-    ```sh
+    ```shell
```
Comment (iosmanthus) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:63]: We may use `sh` instead, `shell` will cancel the code highlight of some editors like vscode.

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 2]
```diff
@@ -45,40 +51,41 @@ To view the Grafana: http://127.0.0.1:3000
...
     git clone https://github.com/pingcap/go-ycsb.git
     ```
 
 2. Build the application from the source.
 
-    ```sh
+    ```shell
```
Comment (iosmanthus) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:69]: Ditto

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 3]
```diff
@@ -45,40 +51,41 @@ To view the Grafana: http://127.0.0.1:3000
...
     make
     ```
 
-3. Load a workload using `go-ycsb` with **10000** keys.
+3. Load a workload using `go-ycsb` with **10000** keys into the TiKV cluster.
 
-    ```sh
+    ```shell
```
Comment (iosmanthus) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:75]: Ditto

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 4]
```diff
@@ -87,18 +94,21 @@ Within python3.5+ REPL environment, you can scan all the keys that just inserted
...
+## Run a workload against TiKV
 
-## Step 4. Run a sample workload.
+### Step 1. Run a sample workload
 
-Go to the source directory of `go-ycsb`, use the following command to run the `workloada` from YCSB benchmark, simulating multiple client connections, each performing mixed 50% read and 50% write operations.
+Enter the source directory of `go-ycsb` and use the following command to run the `workloada` from the YCSB benchmark.
 
-```sh
+`workloada` simulates multiple client connections and performs a mix of reads (50%) and writes per connection.
```
Comment (iosmanthus) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:105]:
```suggestion
`workloada` simulates multiple client connections and performs a mix of reads (50%) and writes (50%) per connection.
```

**Comment by iosmanthus** [comment]
Rest LGTM

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md (hunk 1 above)
Comment (TomShawn) [reply content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:63]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md (hunk 2 above)
Comment (TomShawn) [reply content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:69]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md (hunk 3 above)
Comment (TomShawn) [reply content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:75]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 5]
```diff
@@ -110,96 +120,96 @@ UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 1050
...
 {{< info >}}
-While using `tiup ctl`, an explicit version of the component is needed. In this example, it's v5.1.0.
+The component version should be explicitly specified in the `tiup ctl` command. In the following example, the component version is v5.1.0.
 {{< /info >}}
-1. Increase replicas to the cluster:
 
-    ```sh
+1. Increase the replicas of the cluster to five:
+
+    ```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:192]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 6]
```diff
@@ -110,96 +120,96 @@ UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 1050
...
     tiup ctl:v5.1.0 pd config set max-replicas 5
     ```
 
-2. Stop 2 non-leader nodes simultaneously. In this example, we stop the processes that hold port `20181` and `20182` whose PID is `1009934` and `109941`.
+2. Stop two non-leader nodes simultaneously. In this example, the processes that hold the ports `20181` and `20182` are stopped. The process IDs (PIDs) are `1009934` and `109941`.
 
-   ```sh
+   ```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:198]:
```suggestion
   ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 7]
```diff
@@ -210,11 +220,14 @@ While using `tiup ctl`, an explicit version of the component is needed. In this
...
-2. You can destroy the cluster by:
+## Clean up the test cluster
+
+After experiment 2 is finished, you might need to clean up the test cluster. To do that, take the following steps:
+
+1. Go back to the terminal session that you have just started the TiKV cluster and press <kbd>ctrl</kbd>> + <kbd>c</kbd> and wait for the cluster to stop.
+2. After the cluster is stopped, destroy the cluster using the following command:
 
-    ```sh
+    ```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:231]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 8]
```diff
@@ -110,96 +120,96 @@ UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 1050
...
+In this example, the only one leader in the cluster is stopped. Then the load continuity and cluster health are checked.
 
-1. Go to Grafana dashboard **playground-overview**, the leader distribution is in panel **leader** in row **TiKV**.
+1. Enter the Grafana dashboard **playground-overview**. The leader distribution is shown in the panel **leader** in row **TiKV**.
 
-2. In the example, the local process that open port `20180` hold only one leader in the cluster. Use the following command to stop the process.
+2. In this example, the local process that opens the port `20180` holds only one leader in the cluster. Execute the following command to stop this process.
 
-    ```sh
+    ```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:158]:
```suggestion
    ```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 9]
```diff
@@ -1,34 +1,40 @@
...
-Use `tiup playground` command to launch a 6-node local cluster.
++ [TiUP](https://github.com/pingcap/tiup) is installed (v1.5.2 or later) as described in [TiKV in 5 Minutes](../../tikv-in-5-minutes).
++ [client-py](https://github.com/tikv/client-py) is installed. It is used to interact with the TiKV cluster.
 
-```sh
+### Step 1. Start a six-node cluster
+
+Use the `tiup playground` command to start a six-node local TiKV cluster:
+
+```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:33]:
```suggestion
```sh
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 10]
```diff
@@ -87,18 +94,21 @@ Within python3.5+ REPL environment, you can scan all the keys that just inserted
...
-## Step 4. Run a sample workload.
+### Step 1. Run a sample workload
 
-Go to the source directory of `go-ycsb`, use the following command to run the `workloada` from YCSB benchmark, simulating multiple client connections, each performing mixed 50% read and 50% write operations.
+Enter the source directory of `go-ycsb` and use the following command to run the `workloada` from the YCSB benchmark.
 
-```sh
+`workloada` simulates multiple client connections and performs a mix of reads (50%) and writes (50%) per connection.
+
+```shell
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:107]:
```suggestion
```sh
```

**Review by iosmanthus — APPROVED** [review]
LGTM! PTAL @marsishandsome

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 11]
```diff
@@ -110,96 +120,96 @@ UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 1050
...
 
-2. In the example, the local process that open port `20180` hold only one leader in the cluster. Use the following command to stop the process.
+2. In this example, the local process that opens the port `20180` holds only one leader in the cluster. Execute the following command to stop this process.
 
     ```sh
     kill -STOP $(lsof -i:20180 | grep tikv | head -n1 | awk '{print $2}')
     ```
 
-## Step 7. Check load continuity and cluster health
+## Step 2. Check the load continuity and cluster health on Grafana dashboard
```
Comment (CharLotteiu) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:162]:
```suggestion
### Step 2. Check the load continuity and cluster health on Grafana dashboard
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 12]
```diff
@@ -110,96 +120,96 @@ UPDATE - Takes(s): 20.0, Count: 15799, OPS: 791.1, Avg(us): 19834, Min(us): 1050
...
-2. Stop 2 non-leader nodes simultaneously. In this example, we stop the processes that hold port `20181` and `20182` whose PID is `1009934` and `109941`.
+2. Stop two non-leader nodes simultaneously. In this example, the processes that hold the ports `20181` and `20182` are stopped. The process IDs (PIDs) are `1009934` and `109941`.
 
    ```sh
    kill -STOP 1009934
    kill -STOP 1009941
    ```
 
-## Step 9. Check load continuity and cluster health
+## Step 2: Check the load continuity and cluster health on Grafana dashboard
```
Comment (CharLotteiu) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:203]:
```suggestion
### Step 2: Check the load continuity and cluster health on Grafana dashboard
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 13]
```diff
@@ -210,10 +220,13 @@ While using `tiup ctl`, an explicit version of the component is needed. In this
...
     ```
-## Step 10. Clean up
 
-1. Back to the terminal session that you just started the TiKV cluster and press `ctrl-c` and wait for the cluster to stop.
-2. You can destroy the cluster by:
+## Clean up the test cluster
+
+After experiment 2 is finished, you might need to clean up the test cluster. To do that, take the following steps:
+
+1. Go back to the terminal session that you have just started the TiKV cluster and press <kbd>ctrl</kbd>> + <kbd>c</kbd> and wait for the cluster to stop.
```
Comment (CharLotteiu) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:228]:
```suggestion
1. Go back to the terminal session that you have just started the TiKV cluster and press <kbd>ctrl</kbd> + <kbd>c</kbd> and wait for the cluster to stop.
```

File: content/docs/5.1/reference/architecture/overview.md [hunk 14]
```diff
@@ -42,6 +42,11 @@ Based on the Raft layer, TiKV provides two APIs that clients can interact with:
 | Raw           | A lower-level key-value API to interact directly with individual key-value pairs | Single key    | Your application requires low latency and does not involve multi-key transactions. |
 | Transactional | A higher-level key-value API to provide snapshot isolation transaction           | Multiple keys | Your application requires distributed transactions.                           |
 
+| Concept        |                                                   Description                                                    |
+| -------------- | :--------------------------------------------------------------------------------------------------------------: |
+| **Raft Group** |                  Each replica of a region is called Peer. All of such peers form a raft group.                   |
+| **Leader**     | In every raft group, there is a unique role called leader, are responsible for read/write requests from clients. |
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:48]:
```suggestion
| **Leader**     | In every raft group, there is a unique role called leader, who is responsible for processing read or write requests from clients. |
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 15]
```diff
@@ -64,21 +70,22 @@ On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) t
...
 
-3. Load a workload using `go-ycsb` with **10000** keys.
+3. Load a workload using `go-ycsb` with **10000** keys into the TiKV cluster.
 
     ```sh
     ./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="127.0.0.1:2379" -p tikv.type="raw" -p recordcount=1000000
     ```
 
-    This command will output the following content:
+    The command above will output the following result:
```
Comment (CharLotteiu) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:79]:
```suggestion
    The expected output is as follows:
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 16]
```diff
@@ -45,14 +51,14 @@ To view the Grafana: http://127.0.0.1:3000
...
 {{< info >}}
-Each region contains 3 replicas according to the default configuration.
+Each Region has three replicas according to the default configuration.
 {{< /info >}}
 
-## Step 2. Write data
+### Step 2. Import data to TiKV
 
-On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload of writing data to the TiKV cluster.
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:59]:
```suggestion
Start a new terminal session, and use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload of writing data to the TiKV cluster.
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 17]
```diff
@@ -64,21 +70,22 @@ On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) t
...
     ```
     Run finished, takes 11.722575701s
     INSERT - Takes(s): 11.7, Count: 10000, OPS: 855.2, Avg(us): 18690, Min(us): 11262, Max(us): 61304, 99th(us): 36000, 99.9th(us): 58000, 99.99th(us): 62000
     ```
-## Step 3. Verify the data importing.
 
-Within python3.5+ REPL environment, you can scan all the keys that just inserted by `go-ycsb` and assert the count of them is `recordcount` in the `go-ycsb` command above.
+### Step 3: Verify the data import
+
+Using the Python 3.5+ REPL environment, you can scan all the keys that just are inserted by `go-ycsb` and verify that the key count matches the `recordcount` in the `go-ycsb` command in the previous step.
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:88]:
```suggestion
In this demonstration, the Python 3.5+ REPL environment is used to scan all the keys that just are inserted by `go-ycsb`, and to verify that the key count matches the `recordcount` in the `go-ycsb` command in the previous step.
```

File: content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md [hunk 18]
```diff
@@ -64,21 +70,22 @@ On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) t
...
     ```
     Run finished, takes 11.722575701s
     INSERT - Takes(s): 11.7, Count: 10000, OPS: 855.2, Avg(us): 18690, Min(us): 11262, Max(us): 61304, 99th(us): 36000, 99.9th(us): 58000, 99.99th(us): 62000
     ```
-## Step 3. Verify the data importing.
 
-Within python3.5+ REPL environment, you can scan all the keys that just inserted by `go-ycsb` and assert the count of them is `recordcount` in the `go-ycsb` command above.
+### Step 3: Verify the data import
+
+In this demonstration, the Python 3.5+ REPL environment is used to scan all the keys that just are inserted by `go-ycsb`, and to verify that the key count matches the `recordcount` in the `go-ycsb` command in the previous step.
```
Comment (TomShawn) [inline content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md:88]:
```suggestion
Use the client-py tool to verify the data imported in the last step. Note that the Python 3.5+ REPL environment is required for such verification. It is expected that the key count matches the `recordcount` in the `go-ycsb` command in the previous step.
```

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**